feat: add mergify integration (#1782)
DataDog/integrations-extras: mergify/README.md

# Agent Check: Mergify

## Overview

> This check tracks [Mergify][1] merge-queue statistics.

Reviewer: ```suggestion
This check tracks [Mergify][1] merge queue statistics.
```

## Setup

### Installation

To install the Mergify check on your host:

> 
> 1. Install the [developer toolkit]
> (https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit)
>  on any machine.

Reviewer 2: ```suggestion

1. Install the [developer toolkit][8] on any machine.
```

Author: Fixed.

3. [Download the Datadog Agent][2].

4. Upload the build artifact to any host with an Agent and
 run `datadog-agent integration install -w
 path/to/mergify/dist/<ARTIFACT_NAME>.whl`.

### Configuration

1. Edit the `mergify.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][11] to start collecting your Mergify [metrics](#metrics).

> See the sample [mergify.d/conf.yaml.example][3] file for all available configuration options.

Reviewer 2: ```suggestion
   See the sample [mergify.d/conf.yaml.example][3] file for all available configuration options.
```

> 1. Edit the `mergify.d/conf.yaml` file in the `conf.d/` folder at the root of your [Agent's configuration directory][3] to start collecting your Mergify [metrics](#metrics). See the [

Reviewer 2: Looks like this step may still be under development -- could you please confirm and update?

Author: I re-added the missing sentence.

2. [Restart the Agent][4].

### Validation

Run the [Agent's status subcommand][5] and look for `mergify` under the Checks section.

## Data Collected

### Metrics

See [metadata.csv][6] for a list of metrics provided by this check.

### Service Checks

See [service_checks.json][7] for a list of service checks provided by this integration.

### Events

Mergify does not include any events.

## Troubleshooting

Need help? Contact [Mergify support][1].

[1]: https://mergify.com
[2]: https://app.datadoghq.com/account/settings#agent
[3]: https://github.com/DataDog/integrations-extras/blob/master/mergify/datadog_checks/mergify/data/conf.yaml.example
[4]: https://docs.datadoghq.com/agent/guide/agent-commands/#start-stop-and-restart-the-agent
[5]: https://docs.datadoghq.com/agent/guide/agent-commands/#agent-status-and-information
[6]: https://github.com/DataDog/integrations-extras/blob/master/mergify/metadata.csv
[7]: https://github.com/DataDog/integrations-extras/blob/master/mergify/assets/service_checks.json

> 

Reviewer 2: ```suggestion
[8]: https://docs.datadoghq.com/developers/integrations/new_check_howto/#developer-toolkit
```

Author: Fixed.
